components/Grid.js

```diff
@@ -25,9 +25,12 @@ Box.propTypes = {
   ...propTypes.flexbox,
 };
 
-export const Flex = styled(Box)({
-  display: 'flex',
-});
+export const Flex = styled(Box)(
+  {
+    display: 'flex',
+  },
+  compose(space, layout, flexbox),
```

Comment:
I don't think we were matching the rebass behavior. I'm pretty much sure you can pass `alignItems="center" overflow="hidden"` props in rebass.

Reply:
It matches the rebass behavior from when we removed the dependency. This file was initially just a copy-paste of https://github.com/rebassjs/grid/blob/master/src/index.js

Reply:
Isn't it weird that a div Flex component accepts no CSS flex or div properties?

Reply:
@kewitz It does accept CSS flex properties already, since it inherits `Box` which implements styled-system's `flexbox`. Actually, the properties that you added there are already supported by the component.